src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/Helpers/SystemThemeHelper.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Windows.UI;
+using Windows.UI.ViewManagement;
+using Windows.UI.Xaml;
+
+namespace Uno.Material.Samples.Helpers
+{
+	public static class SystemThemeHelper
+	{
+		/// <summary>
+		/// Get the ApplicationTheme of the device/system
+		/// </summary>
+		public static ApplicationTheme GetSystemApplicationTheme()
+		{
+#if __ANDROID__
+			if ((int)Android.OS.Build.VERSION.SdkInt >= 28)
+			{
+				var uiModeFlags = Android.App.Application.Context.Resources.Configuration.UiMode & Android.Content.Res.UiMode.NightMask;
+				if (uiModeFlags == Android.Content.Res.UiMode.NightYes)
+				{
+					return ApplicationTheme.Dark;
```

Comment:
Don't we have this in Uno ? @davidjohnoliver

Reply:
We might, I just copied it quickly from the Uno.Gallery repo. I will check

Reply:
I don't see it anywhere

Reply:
It's `internal` in Uno.

We may be able to use the `ThemeListener` class from WCT, now that @MartinZikmund has implemented settings for `UISettings`. (See also https://github.com/unoplatform/nventive-private/issues/38)